### PR TITLE
Stub out needed things to make testing with a current user work

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

File: source/OptimaJet.DWKit.StarterApplication/Services/HttpCurrentUserContext.cs [hunk 1]
```diff
@@ -30,18 +30,24 @@ public class HttpCurrentUserContext : ICurrentUserContext
             this.TokenService = tokenService;
         }
 
-        private string AuthType {
-            get {
-                if (authType == null) {
+        private string AuthType
```
Comment (NullVoxPopuli) [inline source/OptimaJet.DWKit.StarterApplication/Services/HttpCurrentUserContext.cs]: I found out ctrl+k, ctrl+d auto-formats in Visual Studio Community Edition
(I switched to VS Community Edition, for debugging, as I haven't figured out a good way to handle that in vim yet)

Comment (NullVoxPopuli) [reply source/OptimaJet.DWKit.StarterApplication/Services/HttpCurrentUserContext.cs]: so, I need to figure out a good way we can get more automated styling of the C# code

**Comment by NullVoxPopuli** [comment]
Next TODO will be to add backend tests to the CI runner.
right now, the ci runner does nothing for backend changes.

File: source/SIL.AppBuilder.Portal.Backend.Tests/Acceptance/Support/TestCUrrentUserContext.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿using OptimaJet.DWKit.StarterApplication.Services;
```
Comment (davidmoore1) [inline source/SIL.AppBuilder.Portal.Backend.Tests/Acceptance/Support/TestCUrrentUserContext.cs]: The filename is TestCUrrentUserContext according to the display here.  Minor but CU should be Cu.

Comment (NullVoxPopuli) [reply source/SIL.AppBuilder.Portal.Backend.Tests/Acceptance/Support/TestCUrrentUserContext.cs]: most def. been havin Visual Studio issues with renaming today

**Pull request merged**